bin/g

```diff
@@ -579,7 +585,24 @@ activate() {
 # Install <version>.
 #
 
-install() {
+install_version() {
+  version=$1
+  dir="$GO_VERSIONS_DIR/$version"
```

Comment:
Before setting this, we have to resolve the `latest` version in case such keyword was used. Same for the set version.

Since we would be giving false positive since `$GO_VERSIONS_DIR/latest` is not an actual thing.

Reply:
i see. i miss this one